File: transformers/hmdb/db/src/org/broadinstitute/translator/db/hmdb/JSON.scala
package org.broadinstitute.translator.db.hmdb

import com.fasterxml.jackson.databind.ObjectMapper;
import com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper
import com.fasterxml.jackson.databind.DeserializationFeature;

object JSON {

  val mapper = new ObjectMapper() with ScalaObjectMapper

  mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
}

File: MoleProAPI/java-play-framework-server/app/transformer/util/JSON.java
package transformer.util;

import com.fasterxml.jackson.annotation.JsonInclude.Include;
import com.fasterxml.jackson.databind.DeserializationFeature;
import com.fasterxml.jackson.databind.ObjectMapper;

public class JSON {

	/**
	 * JSON mapper
	 */
	public static final ObjectMapper mapper = new ObjectMapper();
	
	static {
		mapper.setSerializationInclusion(Include.NON_NULL);
		mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
	}


	private JSON() {
		// static methods only
	}

}
